src/java/org/apache/cassandra/thrift/CustomTThreadPoolServer.java

```diff
@@ -75,16 +78,20 @@ public class CustomTThreadPoolServer extends TServer
     // Server options
     private final TThreadPoolServer.Args args;
 
-    //Track and Limit the number of connected clients
+    // Track and Limit the number of connected clients
```

Comment:
This will make rebasing harder. Yes it doesn't fit our code style, but synthetic changes should be avoided in my opinion.

Reply:
agree, should have mentioned this PR is for observability during our testing only and won't be merged (will close after we've root caused the issue)